### PR TITLE
Dijkstra Algorithm implementation in dart

### LINKED ISSUE #2204: Dijkstra Algorithm
- [x] C
- [x] C++
- [ ] C# - @anshu18107
- [ ] CoffeeScript
- [x] Dart - @Swarnimashukla
- [ ] Go - @raksha009
- [x] Java
- [ ] JS - @abhu-A-J
- [ ] Kotlin - @garimasingh128
- [x] Php - @sukritishah15
- [x] Python
- [ ] Readme - @shwetagurnani
- [x] Ruby
- [ ] Rust - @kichloo
- [x] TypeScript

### PR DESCRIPTION
**Pull request by Swarnimashukla**
Fixes #2204 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added Implementation of Dijkstra Algorithm in dart 

#### Languages Used:
- Dart

#### Files Added:
- Dijkstra_Algorithm.dart

@jainaman224 @raghav-dalmia @singh-shreya6 @asha15

### CODE REVIEW

File: Dijkstra_Algorithm/Dijkstra_Algorithm.dart [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        {
+            min = dist[v];
+            min_index = v;
+        }
+    }
+
+    return min_index;
+}
+
+void printsol(dist , n)
```
Comment (asha15) [inline Dijkstra_Algorithm/Dijkstra_Algorithm.dart:29]: Remove the space before comma

File: Dijkstra_Algorithm/Dijkstra_Algorithm.dart [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        dist[i] = INT_MAX;
+        visited[i] = false;
+    }
+
+    dist[src] = 0;
+
+    for (var count = 0; count < n ; count++)
+    {
+
+        var u = minDistance(dist, visited , n);
```
Comment (asha15) [inline Dijkstra_Algorithm/Dijkstra_Algorithm.dart:56]: remove the space before comma

File: Dijkstra_Algorithm/Dijkstra_Algorithm.dart [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    {
+        dist[i] = INT_MAX;
+        visited[i] = false;
+    }
+
+    dist[src] = 0;
+
+    for (var count = 0; count < n; count++)
+    {
+
```
Comment (asha15) [inline Dijkstra_Algorithm/Dijkstra_Algorithm.dart:55]: remove this line

Comment (Swarnimashukla) [reply Dijkstra_Algorithm/Dijkstra_Algorithm.dart]: Made necessary changes @asha15  please review.

**Comment by asha15** [comment]
Keep going :)

**Comment by kichloo** [comment]
@asha15 Pl review my PR

**Comment by asha15** [comment]
@kichloo sure. will review it

**Pull request merged**